examples/digital_out/write_dig_lines.py

```diff
@@ -0,0 +1,19 @@
+"""Example for generating digital signals.
+
+This example demonstrates how to write values to a digital
+output channel.
+"""
+
+import nidaqmx
+from nidaqmx.constants import LineGrouping
+
+with nidaqmx.Task() as task:
+    data = [True, False, True, False]
+
+    task.do_channels.add_do_chan("Dev1/port0/line0:3", line_grouping=LineGrouping.CHAN_PER_LINE)
+    task.write(data, auto_start=True)
+
+    while not task.is_task_done():
+        pass
+
+    task.stop()
```

Comment:
Sorry for the late change in direction, but the analog output section of https://www.ni.com/docs/en-US/bundle/daqhelp/page/startfunction.html has guidance about when to use auto-start. It sounds like it's only appropriate for on-demand, and only if you want the task to auto-stop. 

Based on this, we ought to use explicit start like the C examples do.

```suggestion
    task.start()
    task.write(data)
    task.stop()
```

Reply:
It's alright, thanks for the suggestion. It's updated as per suggestion.